metrics/ap_pipe.yaml

```diff
@@ -22,3 +22,26 @@ ApPipeMemory: &Memory
         - task
         - monitoring
 
+ApFakesCompletenessMag20t22:
+    description: Completeness of recovered fakes in the magnitudes range 20-22.
+    unit: '%'
```

Comment:
Is this metric defined in any document or something? If so, consider providing a `reference` entry.